src/message.py

```diff
@@ -0,0 +1,59 @@
+# _*_ coding: utf-8 _*_
+"""Classes for the response and the request messages."""
+from __future__ import unicode_literals
+
+HTML_PROTOCOL = 'HTTP/1.1'
+
+RESPONSE_CODE = {
+    200: '200 OK',
+    400: '400 Bad Request',
+    405: '405 Method Not Allowed',
+    500: '500 Internal Server Error'
+}
+
+CRLF = '\r\n'
+
+
+class Response(object):
+    """Form a correct http response."""
+
+    def __init__(self, code, body=None):
+        """Initialize the response.  Pass in error code."""
+        self.code = code
```

Comment:
since eventually this code is going to be used as a key into the RESPONSE_CODE dict, you might want to validate the incoming value to make sure it's in there.  Otherwise you run the risk of a key error on line 29 below.